packages/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -36,6 +36,9 @@ import {
   getNextColor,
   getColorFromToken,
 } from '../../utilities/index';
+import { lazy } from 'react';
+
+const Legends = lazy(() => import('../Legends/Legends'));
```

Comment:
why not use ../Legends/index ?

Reply:
This is Removed, as  this not needed for now as legends are not lazy loaded.